### PR TITLE
adq patch selector for each area in ui

### PR DESCRIPTION
**Pull request by rtei-pw**

### CODE REVIEW

File: src/libs/antares/study/area/list.cpp [hunk 1]
```diff
@@ -332,8 +332,7 @@ bool saveAreaAdequacyPatchIniFile(const Area& area, const Clob& buffer)
 {
     IniFile ini;
     IniFile::Section* section = ini.addSection("adequacy-patch");
-    bool bUseAdequacyPatch = false; //adq: this should be later replaced with area.bUseAdequacyPatch
-    section->add("use-adequacy-patch", static_cast<bool>(bUseAdequacyPatch));
+    section->add("use-adequacy-patch", static_cast<bool>(area.bUseAdequacyPatch));
```
Comment (hugo-antoine-rtei) [inline src/libs/antares/study/area/list.cpp]: static cast not required see [here](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/532#discussion_r793292520)

**Review by hugo-antoine-rtei — CHANGES_REQUESTED** [review]
I propose:
- Rebase once PR #4 is merged (format of area.cpp will be conflicting) - I'll wait your approval
- clang-format the files you changed

**Comment by rtei-pw** [comment]
> I propose:
> 
> * Rebase once PR #4 is merged (format of area.cpp will be conflicting) - I'll wait your approval
> * clang-format the files you changed

PR #4 is merged; clang format modified

**Review by hugo-antoine-rtei — APPROVED** [review]
looks good !

**Pull request merged**